src/Raven.Server/Documents/Indexes/IndexingStatsAggregator.cs

```diff
@@ -85,7 +85,8 @@ private IndexingPerformanceStats CreateIndexingPerformanceStats(bool completed)
                 SuccessCount = Stats.MapSuccesses + Stats.MapReferenceSuccesses,
                 FailedCount = Stats.MapErrors + Stats.MapReferenceErrors,
                 OutputCount = Stats.IndexingOutputs,
-                DocumentsSize = new Size(Stats.DocumentsSize)
+                DocumentsSize = new Size(Stats.DocumentsSize),
+                AllocatedBytes = Stats.AllocatedBytes
```

Comment:
Is this displayed in the Studio?

Reply:
it is not displayed, but now it should be exposed to the studio.